### ISSUE TITLE
Media keys not working on Ubuntu 22.04

### ISSUE THREAD

**Issue by mvanorder** [Mastermindzh/tidal-hifi#319]
Play/pause, next, previous don't work on the Tidal app.  They do however work on other things(i.e. YouTube videos in the browser).  While Tidal is playing if I try to pause with the keyboard or my headset it will instead play whatever else might be in the background that does work with the media keys and the Tidal app will remain unaffected.

**Comment by Mastermindzh** [comment]
Hey,

Few questions:

- do you have media keys enabled in settings?
- does media show up in any dropdown/up from your system?
- what DE?

**Comment by mvanorder** [comment]
I unchecked "Disable hardware built-in media keys" in the Advanced tab of the settings yesterday.  This had no affect even after closing and restarting the Tidal app.  However today(after a reboot) I read your message and check the media in gnome to make sure it's showing the song there and now everything is working(keyboard and headset can control Tidal).

Apologies, I thought I had exhausted my options for troubleshooting. I can only assume somehow something wasn't working in gnome yesterday which made it so that changing the setting had no affect until after a reboot.
